# Request
string uuid
---
# Response
bool success